WordPress/Classes/ViewRelated/Reader/ReaderRetryFailedImageView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ReaderRetryFailedImageView.swift
+//  WordPress
+//
+//  Created by Paul Von Schrottky on 10/5/19.
+//  Copyright © 2019 WordPress. All rights reserved.
+//
+
+import UIKit
+import Gridicons
+
+protocol ReaderRetryFailedImageDelegate: AnyObject {
+    func didTapRetry()
+}
+
+fileprivate class LinkTextView: UITextView {
+    override func selectionRects(for range: UITextRange) -> [UITextSelectionRect] {
+        return []
+    }
+
+    override func caretRect(for position: UITextPosition) -> CGRect {
+        return CGRect.zero.offsetBy(dx: .greatestFiniteMagnitude, dy: .greatestFiniteMagnitude)
+    }
+}
+
+class ReaderRetryFailedImageView: UIView {
```

Comment:
> This means that we created instances of this class for every image even if they're not needed. Would you please let me know why it behaves like this?

Thanks for catching this - it's because I accessed the lazy property in:
`self.retryImageView.removeFromSuperview()`
I'll update this to ensure it's handled properly.

> Also, the post has 3 images only but there were 6 recorded lines in the log. I didn't investigate further why that is.
I traced this back to:
https://github.com/wordpress-mobile/WordPress-iOS/blob/a9d3de84faae974e8508be6f995bd7eaad8af6d5/WordPress/Classes/ViewRelated/Views/WPRichText/WPTextAttachmentManager.swift#L110
which is called multiple times. This is the same in `develop`, let me know if you'd like me to dig further.

Reply:
> I'll update this to ensure it's handled properly.

Thanks! This is good enough for me. As for the `enumerateAttachments()`, let's leave that alone.